### ISSUE TITLE
Support for npm v6

### ISSUE THREAD

**Issue by MercierCorentin** [roderickhsiao/react-aspect-ratio#40]
I'm receiving these warnings when I install the npm package:
```
npm WARN notsup Unsupported engine for react-aspect-ratio@1.0.41: wanted: {"npm":"^3.0.0 || ^5.0.0"} (current: {"node":"10.15.3","npm":"6.12.0"})
npm WARN notsup Not compatible with your version of node/npm: react-aspect-ratio@1.0.41
```
What is written in my `package.json`:
```json
"react-aspect-ratio": "^1.0.41",
```
Will the package be supported for npm V6?

Note: I might interpret this wrong as I don't know much about npm.

**Comment by roderickhsiao** [comment]
Hey thanks for reporting!

When I create the package, npm v6 is not yet there :) thats why its not in the range.
The package itself should support npm v6, I just published a new package 1.0.42 without the npm restriction. let me know if it works.

**Comment by MercierCorentin** [comment]
It works, thanks!
